templates/angular-material/Program.cs

```diff
@@ -34,8 +34,8 @@
             if (context.File.Name == "index.html")
             {
                 // more info on this header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#preventing_caching
-                context.Context.Response.Headers.Add("Cache-Control", "no-store, max-age=0");
-                context.Context.Response.Headers.Add("Expires", "-1");
+                context.Context.Response.Headers.Append("Cache-Control", "no-store, max-age=0");

```

Comment:
I'm familiar with the warning this is addressing, but is `Append` appropriate here or should we be using the indexer to set (overwrite) the values if they already exist in the dictionary?